tests/network/vmi_networking.go

```diff
@@ -284,15 +284,12 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 		})
 	})
 
-	Context("VirtualMachineInstance with disabled automatic attachment of interfaces", func() {
+	Context("VirtualMachineInstance with no networks,interfaces and disabled automatic attachment of interfaces", func() {
 		It("[test_id:1774]should not configure any external interfaces", func() {
 			By("checking loopback is the only guest interface")
-			autoAttach := false
+			// Create VMI with no interfaces and networks
```

Comment:
```suggestion
			// Create VMI with no interfaces and networks
```
nit: I see this comment as a distraction. If required, I'd add it to the `By` string (which I find distracting as well, but that's outside this PR)